assets/js/components/BatchEdit/About/Confirmation.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import {
+  CONTROLLED_METADATA,
+  UNCONTROLLED_METADATA,
+  PHYSICAL_METADATA,
+  RIGHTS_METADATA,
+  IDENTIFIER_METADATA,
+  OTHER_METADATA,
+} from "../../../services/metadata";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import UIFormField from "../../UI/Form/Field";
+import UIFormInput from "../../UI/Form/Input";
+import { toastWrapper } from "../../../services/helpers";
+
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+
+const addWrapperCss = css`
+  border: 1px solid green;
+`;
+
+const removeWrapperCss = css`
+  border: 1px solid red;
+`;
+
+const confirmationNote =
+  'NOTE: This will affect all works currently selected. Please proceed with extreme caution. To execute this change, type "I understand"';
+
+const deleteMetadata = {
```

Comment:
@katdivyareddy10 Can you update this to read `mockDeleteMetadata` or something similar so we know it's dummy data?